api-module/src/main/java/org/devridge/api/domain/member/entity/Member.java

```diff
@@ -0,0 +1,47 @@
+package org.devridge.api.domain.member.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.devridge.api.AbstractTimeEntity;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "member")
+@Getter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
@NoArgsConstructor 사용 시 접근 제한을 걸어두시면 좋습니다.
```suggestion
@NoArgsConstructor(access = AccessLevel.PROTECTED)
```
https://github.com/techeer-TIL-group/yu-heejin/blob/main/Spring/Entity%EC%97%90%EC%84%9C%20%EA%B8%B0%EB%B3%B8%20%EC%83%9D%EC%84%B1%EC%9E%90%EC%97%90%20Protected%EB%A5%BC%20%EC%84%A4%EC%A0%95%ED%95%98%EB%8A%94%20%EC%9D%B4%EC%9C%A0.md

Reply:
추가적으로 @SqlDelete를 사용하시면 저희 soft delete 기능을 쉽게 이용하실 수 있습니다.

Reply:
`@NoArgsConstructor` 제거하고 직접 생성자 선언했습니다. (위 글 참조)

soft delete는 관련 개념 학습후에 추가하겠습니다. ㅠㅠ

Reply:
soft delete는 저희가 이전에 합의한대로 is_deleted를 true로 바꾸는 것이라 어렵지 않습니다!

Reply:
넵 삭제 로직 soft delete 되게 수정하겠습니다!